FeatureToggle.Infrastructure/Migrations/20241113075050_FeatureMig1.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace FeatureToggle.Infrastructure.Migrations
+{
+    /// <inheritdoc />
+    public partial class FeatureMig1 : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.EnsureSchema(
+                name: "FeatureDB");
+
+            migrationBuilder.CreateTable(
```

Comment:
Migration is not required for the tables inside FeatureContext

Reply:
noted